### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#470] [Esami & Esoneri M-Z]
### Traccia

<img width="611" alt="image" src="https://github.com/user-attachments/assets/510dc901-62cc-4be3-a32f-49dd59cd0375">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/c2cfbfd1-1635-484e-aa31-8a2e855f5f4b)
